README.md

```diff
@@ -3,6 +3,11 @@ extension-tao-booklet
 
 An extension for TAO to create test booklets (publishable in MS-Word and PDF along with Answer Sheets)
 
+## Warning
+Due to the move of core libs to `npm` repositories, some code might not work on legacy browsers.
+Especially for code that use to rely on polyfills, like for the `Promise`. The polyfills are now linked only when the code is bundled, and are not reachable anymore in development mode.
+For that reason, and because `wkhtmltopdf` is not supporting ES2015 and requires polyfills, the generation of PDF only works with bundled version (aka production mode).   
```

Comment:
(when the constant `DEBUG_MODE` is set to `false` in `config/generis.conf.php`)

Reply:
sure, I'll add more about it in the readme